documentation/docs/connectors/balancer.md

```diff
@@ -27,7 +24,7 @@ When creating Hummingbot Gateway, it picks up the Ethereum settings in the globa
 
 ![](/assets/img/connect-ethereum.gif)
 
-## Install Hummingbot Gateway
```

Comment:
Gateway is the name of our other project/software. So it should be in uppercase.